src/python/ensembl/io/genomio/manifest/check_integrity.py

```diff
@@ -412,7 +423,11 @@ def __init__(self, manifest_file: Path, brc_mode: bool = False, ignore_final_sto
         self.set_ignore_final_stops(ignore_final_stops)
         self.errors: List[str] = []
 
-    def add_errors(self, *errors: str) -> None:
+    def add_error(self, error: str) -> None:
+        """Store the given error in the list."""
+        self.errors.append(error)
+
+    def add_errors(self, errors: List[str]) -> None:
```

Comment:
I'm usually against variable/function names that differentiate by one `s`, as it can be hard to stop and complex to debug. If you didn't like the previous implementation, you can always opt for something in between the two:
```
    def add_errors(self, errors: Union[str, List[str]]) -> None:
        """Store the given errors in the list."""
        if isinstance(errors, str):
            self.errors.append(error)
        else:
            self.errors += errors
```